assets/main.css

```diff
@@ -93,7 +93,6 @@ header {
 section {
   width: 100vw;
   min-height: 800px;
```

Comment:
Does this need to be so tall? I feel like we are getting some odd length issues on small screens (i.e. this could be much more compact.)
Probably doesn't belong in this PR so I can open an issue